src/components/AttachmentPicker/index.js

```diff
@@ -41,6 +41,9 @@ class AttachmentPicker extends React.Component {
                         // Cleanup after selecting a file to start from a fresh state
                         this.fileInput.value = null;
                     }}
+
+                    // Prevent parent's onclick event from firing
```

Comment:
ideally this would say why - something like 

>We are stopping the event propagation because triggering the `click()` on the hidden input causes the event to unexpectedly bubble up to anything wrapping this component e.g. Pressable

That is better since someone will look at this later and have some idea about why the click event propagation needs to be stopped.